l5kit/l5kit/sampling/agent_sampling.py

```diff
@@ -153,10 +147,10 @@ def generate_agent_sample(
         "history_positions": history_coords_offset,
         "history_yaws": history_yaws_offset,
         "history_availabilities": history_availability,
-        "world_to_image": world_to_image_space,
-        "centroid": agent_centroid,
-        "yaw": agent_yaw,
-        "extent": agent_extent,
+        "world_to_image": raster_from_world,
```

Comment:
we should create a `raster_from_world` with the same value and start deprecating `world_to_image`